models/pod.go

```diff
@@ -187,6 +196,28 @@ func (pod Pod) HasIstioSidecar() bool {
 	return len(pod.IstioContainers) > 0
 }
 
+// HasAnyAmbient returns true if any pod is labeled as ambient
+func (pods Pods) HasAnyAmbient() bool {
+	if len(pods) > 0 {
+		for _, p := range pods {
+			if p.HasAmbient() {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+// HasAmbient returns true if the pod is labeled as ambient-type
+func (pod *Pod) HasAmbient() bool {
+	return pod.Annotations[ambientAnnotation] == "enabled"
+}
+
+// IsWaypoint returns true if the pod is labeled as ambient-type
```

Comment:
Comment here is the same as line 211, a bit confusing.

Reply:
That's right, @Xunzhuo , updated  comment